app/config.json

```diff
@@ -0,0 +1,3 @@
+{
+        "path1": "pointing/to/path/where/workspaces/are/stored"
```

Comment:
I would not suggest the variable names as path1. It creates ambiguity among end-users. 
Please change it to something understandable like workspace_dir or workspace_path

Reply:
Though JSON easy way to store configurations, It does not support comments inside the file. 
Could you please change it YAML. 
within YAML you can have the flexibility to add sections and comments too. 
You can always load a YAML file to JSON by 
```
import yaml

with open("example.yaml", 'r') as stream:
    try:
        print(yaml.safe_load(stream))
    except yaml.YAMLError as exc:
        print(exc)
```